lib/puppet_forge/v3/base/paginated_collection.rb

```diff
@@ -5,12 +5,15 @@ class Base
       # Enables navigation of the Forge API's paginated datasets.
       class PaginatedCollection < Array
 
+        LIMIT = 10
```

Comment:
I would prefer that this is commented. Just good policy for "magic number" constants.